src/fides/api/ctl/migrations/versions/317e6197c76a_reduce_experience_and_config_duplication.py

```diff
@@ -0,0 +1,250 @@
+"""reduce experience and config duplication
+
+Revision ID: 317e6197c76a
+Revises: b8248ce6b8a1
+Create Date: 2023-06-05 23:24:43.174826
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "317e6197c76a"
+down_revision = "b8248ce6b8a1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_index(
+        "ix_privacyexperiencehistory_experience_config_history_id",
+        table_name="privacyexperiencehistory",
+    )
```

Comment:
Note, lots of rearranging had to be done in this migration to make sure we were dropping indices/constraints before tables.

Reply:
better now than when these tables are actually in use by customers 😄